app/backend/src/couchers/utils.py

```diff
@@ -96,6 +97,19 @@ def get_coordinates(geom):
     return (shp.y, shp.x)
 
 
+def timezone_at_coordinate(lat, lng):
+    """
+    Returns the timezone name corresponding the lat, lng input, or None if the input does not correspond to a timezone or is invalid.
+    The timezone names are defined in this timezone database:
+    https://www.iana.org/time-zones
+    """
+    tf = TimezoneFinder(in_memory=True)
+    try:
+        return tf.timezone_at(lat=lat, lng=lng)
+    except ValueError as ve:
+        print("Error: timezone_at_coordinate():", ve)
```

Comment:
Please don't `print`, you can `logger.debug` if you wish.

Also please explicitly `return None`, it's just cleaner that way!